modules/configuring-machine-pool-disk-volume-ocm.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+ifdef::openshift-rosa[]
+[id="configuring_machine_pool_disk_volume_ocm{context}"]
+= Configuring machine pool disk volume using OpenShift Cluster Manager
+endif::openshift-rosa[]
+.Prerequisite for day 1
+* You have the option to select the sizing for the default machine pool during cluster installation.
+
+.Procedure for day 1
+
+. Navigate to {cluster-manager-url} to create your cluster.
+
+. Navigate to *Cluster Settings*.
+
+. Navigate to *Node pools*.
+
+. Under the *Advanced machine pool settings*, select the desired *Root disk size*.
+
+. Select *Next* to continue creating your cluster.
+
+.Prerequisite for day 2
+* You have the option to select the sizing for the new day 2 machine pool after the cluster has been installed.
+
+.Procedure for day 2
+
+. Navigate to {cluster-manager-url} and select your cluster.
+
+. Navigate to *Cluster Settings*.
+
+. Navigate to *Node pools*.
+
+. Under the *Advanced machine pool settings*, select the desired *Root disk size*.
+
+. Select *Next* to save cluster settings.
```

Comment:
The text should modify with following steps

- Click *Add machine pool*.
- Select desired *Root disk size*.
- Click *Add machine pool* to create the new machine pool.